hack/devtools/platform_msi.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# This script creates a platform Identities to use for local development
+# The script reads the operators file and creates platform identities for each operator
+
+prompt_for_operators_file() {
+    local operatorsFilePath
+    read -p "Please provide the absolute path of operators file including the file name?" operatorsFilePath
```

Comment:
Cool, that addresses my question about the role assignments and works for me. Did you see my code suggestion to change that question mark to a colon though?

Reply:
Yes, I will fix it once I have @cadenmarchese replies to my comment about using env var. If we have to use env var instead of file, then that line will be removed.